src/view/com/notifications/FeedItem.tsx

```diff
@@ -451,8 +451,9 @@ let FeedItem = ({
             <TimeElapsed timestamp={item.notification.indexedAt}>
               {({timeElapsed}) => (
                 <>
-                  <Text style={[a.ml_xs, pal.textLight]}>&middot;</Text>
-                  <Text style={[a.ml_xs, pal.textLight]} title={niceTimestamp}>
+                  {/* make sure there's whitespace around the middot -sfn */}
```

Comment:
i think you can `<Text style={[pal.textLight]}>{' '}&middot;{' '}</Text>` to make it explicit if you'd like

Reply:
I wrote that - prettier removed it!